### DISCUSSION TITLE
X selections in kitty process

### DISCUSSION THREAD

**Discussion by ghost** [kovidgoyal/kitty#4893] [Q&A] [answered]
Running `pass` commands directly in a terminal works and copies the password to the clipboard. However, when creating a new process, it _usually_ doesn't work i.e.  `kitty pass -c Example/account` or any command with a piped `xclip -selection c`

I'm sure there is a simple explanation for this, I would just like a way to get text copied to the clipboard when starting a new kitty instance. 

I also see that there is a [kitten](https://github.com/kovidgoyal/kitty/issues/1222) for password auto-type, which I could adapt to this case. 

EDIT: Removed non-essential information to the issue.

**Comment by kovidgoyal** [comment]
In X clipbaord contents are tied to the process. If the process that
copies to the clipboard dies, you can longer paste whatever it copied.
To get around this install a clipboard manager.

**Reply by ghost** [reply]
I see, thanks. Would it be possible to send the kitty process to the background for a discrete period of time (like 45 seconds after the command completes) so the programs output can be pasted?

**Comment by kovidgoyal** [comment, marked as answer]
On Tue, Mar 29, 2022 at 10:52:35PM -0700, Carson Surmeier wrote:
> I see, thanks. Would it be possible to send the kitty process to the background for a discrete period of time (like 45 seconds after the command completes) so the programs output can be pasted?

That would be a function of your window manager, iconifying the window.
Use a window manager that is scriptable/controllable.
